CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -182,6 +183,46 @@ private void AnalyseObjectForBinding(JavascriptObject obj, bool analyseMethods,
                 return;
             }
 
+            var dict = obj.Value as IDictionary;
+            if (dict != null)
+            {
+                // in case of dictionary, add each key/value pair as read only property of obj                              
+                foreach (var key in dict.Keys)
+                {
+                    var name = key.ToString();
+                    var value = dict[key];                    
```

Comment:
Prefer not to use `value` as a variable name, whilst it's not a reserved work, it does have special meaning in some contexts and I'd prefer to avoid using it.